spec/models/contact_email_spec.rb

```diff
@@ -21,14 +21,19 @@
     end
   end
 
+  # rubocop:disable RSpec/SubjectStub
   it 'sends a confirmation email' do
+    allow(contact_email).to receive(:saved_change_to_attribute?).with(:email).and_return(true)
```

Comment:
This is probably a bad idea, but I cannot get the `saved_change_to_attribute?(:email)` to return true just by updating the `contact_email.email`. Anyone have ideas?

Reply:
decided in standup yesterday that we're fine with this, i think